app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```

Comment:
@nozomi-matsuoka 
Let's try to create methods for log in.
Create a method here for sessions:
```
def log_in(user)
  session[:user_id] = user.id
end
```

Then you can use this method in SessionsController : 
https://github.com/framgia831/rails-elearning-nozomi/pull/1/files#diff-d5241d488259f32ecbe2f636133e5ddaR6